### ISSUE TITLE
Currency format

### ISSUE THREAD

**Issue by alexjeen** [luads/php-xbase#45]
You can easily add currency format, this works for us:

    const DBFFIELD_TYPE_CURRENCY = 'Y';    // Currency

    public function getCurrency($columnName)
    {
        $s = $this->choppedData[$columnName];

        $s = unpack('q', $s);

        if ($s) {
            return $s[1] / 10000;
        }

        return 0;
    }

**Comment by alexjeen** [comment]
You can also use:

    $s = unpack('l', $s);

It results in the same.

**Comment by hisamu** [comment]
Do you have it working @alexjeen? Would you mind opening a PR?

**Comment by alexjeen** [comment]
Hi there @hisamu, yes we've verified that it works. However we only use the reading part of this library. 

I don't have time to make a PR in the coming weeks for the writing part (and to do some testing for the writing part).

This is the test file we have used to check if it actually works, its a table with a single column and row.

[test.dbf.zip](https://github.com/hisamu/php-xbase/files/1051486/test.dbf.zip)
